app/views/community/index.html.erb

```diff
@@ -12,7 +12,11 @@
   <h2> Mailing List</h2>
 
   <p>
-    Questions or comments for the Git community can be sent to the mailing list by using the email address <a href="mailto:git@vger.kernel.org">git@vger.kernel.org</a>. <strong>Bug reports for git should be sent to this mailing list.</strong>
+    General questions or comments for the Git community can be sent to the mailing list by using the email address <a href="mailto:git@vger.kernel.org">git@vger.kernel.org</a>. 
+  </p>
+  
+  <p>
+    <strong>If you wish to report any possible bug for git, please use this mailing list as well.</strong>
```

Comment:
```suggestion
    <strong>If you wish to report any possible bug for Git, please use this mailing list as well.</strong>
```

While we're at it, let's implement git/git@2de9b71138 (Documentation: the name of the system is 'Git', not 'git', 2013-01-21)